CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 CHANGELOG for Sulu
 ==================
 
+* dev-develop
+    * BUGFIX      #3217 [WebsiteBundle]       Use RedirectController for route redirects (Fix for #2107)
```

Comment:
Sorry, I didn't mean that you should link the other PR here, we never do that... Can you please remove it again?